### ISSUE TITLE
ResNet50 from tf.keras will not fine-tune, but the one from base keras will

### ISSUE THREAD

**Issue by monktastic** [keras-team/keras#8065]
- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [X] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [X] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

I'm excluding the code where I load and preprocess (imagenetutils.preprocess_input). My data is the Kaggle cats-and-dogs data.

```
from tensorflow.contrib.keras.api.keras import backend as K
...

net = ResNet50(include_top=False, weights='imagenet', input_shape=(224, 224, 3))
for layer in net.layers:
    layer.trainable = False
 
output = Flatten(name='my_flatten')(net.output)
output = Dense(num_categories, activation='softmax', name='my_dense')(output)
sc1 = Model(inputs=net.inputs, outputs=output, name='sc1')
sc1.compile(
    loss=keras.losses.categorical_crossentropy,
    optimizer=keras.optimizers.Adam(lr=1e-4),
    metrics=[metrics.categorical_accuracy])
```

This never gets beyond 50-60% accuracy. VGG16 quickly gets to 98%+. If I remove "tensorflow.contrib.keras.api." from the start of all imports, this also gets to 98%+. So something is definitely different between tf and base Keras. I'm just not sure what.
